pkg/store/sandbox/status.go

```diff
@@ -38,13 +38,13 @@ import (
 // |      +------+         |         |           |
 // |      |      |  READY  <---------+           |
 // |      +------>         |         |           |
-// |             +----+----+         |           |
-// |                  |              |           |
-// |                  | Stop/Exit    |           |
-// |                  |              |           |
-// |             +----v----+         |           |
-// |             |         <---------+      +----v----+
-// |             | NOTREADY|                |         |
+// |      |      +----+----+         |           |
+// |      |           |              |           |
+// |      | Reset     | Stop/Exit    |           |
+// |      |           |              |           |
+// |      |      +----v----+         |           |
+// |      |      |         <---------+      +----v----+
+// |      +------+ NOTREADY|                |         |
```

Comment:
Not sure I follow what the change to this chart is doing?

Reply:
I added a transition from the not ready state (stopped) back to ready (running).
It does look pretty confusing though, since the `Ready --> Ready` transition by way of `PortForward` convolutes the new arrow.